src/wallet3/default_daemon_comms.cpp

```diff
@@ -443,10 +467,50 @@ namespace wallet
     return fut;
   }
 
+  std::future<std::string>
+  DefaultDaemonComms::ons_names_to_owners(const std::string& name_hash, const uint16_t type)
+  {
+    auto p = std::make_shared<std::promise<std::string> >();
+    auto fut = p->get_future();
+    auto req_cb = [p=std::move(p)](bool ok, std::vector<std::string> response)
+    {
+
+      oxenc::bt_dict_consumer dc{response[1]};
+
+      if (not dc.skip_until("result"))
+      {
+        auto reason = dc.consume_string();
```

Comment:
This logic doesn't seem right:

If we did *not* find the key `"result"` then read the value of the next pair in the dict as a string and return it as an error value.